crates/cli/src/subcommands/list.rs

```diff
@@ -17,12 +18,7 @@ pub fn cli() -> Command {
                 .required(true)
                 .help("The identity to list databases for"),
         )
-        .arg(
-            Arg::new("server")
-                .long("server")
-                .short('s')
-                .help("The nickname, host name or URL of the server from which to list databases"),
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.